requirements/jax/env_jax.cpu.yml

```diff
@@ -1,8 +1,10 @@
 dependencies:
   - pip:
     - -f https://storage.googleapis.com/jax-releases/jax_releases.html
-    - jax[cpu]<0.3.25
+    - jax==0.3.24
     # 0.3.25 contains breaking change not compatible with haiku
     # https://github.com/deepmind/dm-haiku/issues/565
     - dm-haiku
-    - optax
+    - optax==0.1.5
```

Comment:
Don't pin to old libraries; it causes instabilities

Reply:
fixed, updated jax to the latest version